assignment/Module6/TeacherIUTrigger.apxt

```diff
@@ -0,0 +1,14 @@
+trigger TeacherIUTrigger on Contact (before insert, before update) {
+	List<Contact> contacts;
+    if(Trigger.isInsert){
+        contacts = Trigger.New;
+    } else {
+        contacts = Trigger.Old;
```

Comment:
What if we `upsert` some teacher records . Please test this case. Use `Trigger.New` here.